egs2/chime2/enh1/local/cstr_ndx2flist.pl

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env perl
```

Comment:
Ditto.

Reply:
No, this script is modified from the CHiME4 one and thus different. I just copied this script from Kaldi's recipe.